core/vm/instructions.go

```diff
@@ -30,6 +32,9 @@ import (
 var (
 	bigZero                  = new(big.Int)
 	tt255                    = math.BigPow(2, 255)
+	OvmSLOADMethodId         = hashSha3([]byte("ovmSLOAD()"))[0:4]
+	OvmSSTOREMethodId        = hashSha3([]byte("ovmSSTORE()"))[0:4]
+	OvmContractAddress       = common.FromHex(os.Getenv("EXECUTION_MANAGER_ADDRESS"))
```

Comment:
Will this throw if `EXECUTION_MANAGER_ADDRESS` isn't defined? If not, we'll probably want to do some reasonable config checking and throw a descriptive error right away. Doesn't necessarily need to be in _this_ PR, but I'd add a ticket for it if nothing else.

Reply:
Yeah, good call! I'll add a panic on startup if `EXECUTION_MANAGER_ADDRESS` isn't set.

Reply:
Alright, so after looking into this some more I think this one _may_ warrant it's own ticket. Currently the deploy process is:

1. Boot up Geth without setting `EXECUTION_MANAGER_ADDRESS`.
2. Deploy the contract using the `ExecutionManager` deployment script.
3. Restart Geth but this time set `EXECUTION_MANAGER_ADDRESS`.

If we panic on startup if `EXECUTION_MANAGER_ADDRESS` isn't set then step 1 will cause a panic 😕 

Another option:

We could set the contract code of the `ExecutionManager` and set the initial contract state in the [genesis file](https://github.com/op-optimism/go-ethereum/blob/mf-override-call/etc/optimism.json) and then hardcode the address into the source. This would also require figuring out how to get all of the storage values in the deployed `ExecutionManager` so we could put them in the genesis file. I'm not sure how much time we want to put into this vs how much value we'll get out of it. Let me know what y'all think and/or let's chat more about this at sprint planning! 🏃